src/scss/_grid.scss

```diff
@@ -12,15 +12,26 @@
 		display: grid;
 		box-sizing: border-box;
 		min-height: 100vh;
-		min-width: 100%;
+		max-width: $_o-layout-container-max-width;
+		margin: auto;
 		grid-template-columns: 100%;
 		grid-template-rows: auto 1fr auto;
 		grid-column-gap: $_o-layout-gutter;
+		padding: 0 (oGridGutter() * 2); // To align with o-header-services.
 		grid-template-areas:
 			"header"
 			"main"
 			"footer";
 	}
+
+	.o-layout__footer,
+	.o-layout__header {
+		// Position to the left of the viewport.
+		// Relative to the center aligned grid.
+		position: relative;
+		width: 100vw;
+		left: #{calc((100vw - 100%) * -0.5)};
```

Comment:
So this is happening because the grid doesn't span the whole page, right? Why did we remove the five columns again?

Reply:
Yeah, it's to remove the mega hack which calculated in SCSS a px value for the main content area. We either need a wrapper around the main content and sidebar, or we set the whole of `.o-layout` to the max width of the container and then make the header and footer "break out" of the container.

Reply:
Ah, and the mega hack is bad because if the main content area is set in px it can't expand when the sidebar is deleted.